src/test/java/com/dabsquared/gitlabjenkins/trigger/handler/PendingBuildsHandlerTest.java

```diff
@@ -230,15 +236,17 @@ private PushHook pushHook(int projectId, String branch, String commitId) {
                 .withAfter(commitId)
                 .withRepository(new Repository())
                 .withProject(ProjectBuilder.project().withNamespace("namespace").build())
-                .withCommits(Arrays.asList(
-                        CommitBuilder.commit().withId(commitId).withAuthor(user).build()))
+                .withCommits(Collections.singletonList(
+                        commit().withId(commitId).withAuthor(user).build()))
                 .withRepository(repository)
                 .withObjectKind("push")
-                .withUserName("username")
+                .withUserName("User")
+                .withUserUsername("username")
+                .withUserEmail("user@gitlab.com")
```

Comment:
I think these are the tests we need for this PR I guess, which I have added/updated.